lib/scss/inc/_base.scss

```diff
@@ -40,22 +40,44 @@ a code {
   display: inline-block;
   background-color: $pre-bg;
   padding: 2px 8px;
-  font-size: $font-size-base - 2;
+  font-size: $font-size-base - 1.6;
```

Comment:
Fractional pixels value won't work in browser (most of the time). Depends on the browser, it'll round the value to the closest integer.  With px, it's always better to use a full value.

Reply:
Thanks @rushenn, will change the value of this back to `2` then.